styles/utility.css

```diff
@@ -350,18 +355,60 @@ div.contact-me {
 .text-group {
   width: 100%;
   text-align: center;
+  margin: 12px;
 }
 
-.text-group textarea {
+textarea {
   width: 100%;
+  height: 114px;
+  border-radius: 4px;
+  font-family: 'inter', serif;
+}
+
+.text-group textarea {
+  width: 97%;
   user-select: none;
   outline: none;
   padding: 8px;
-  margin-bottom: 24px;
+  margin-bottom: 8px;
+  border: 1px solid #60c095;
+  border-radius: 4px;
 }
 
 .btn.btn-form-submit {
   background-color: #3db682;
   color: white;
+  width: 129px;
+  height: 48px;
+  font-weight: bold;
+  font-size: 17px;
   text-align: center;
+  border-radius: 6px;
+}
+
+.footer-img {
+  position: absolute;
+  bottom: 3.5%;
+  right: 0;
+}
+
+::placeholder {
+  font-size: 18px;
+  padding-left: 14px;
+  color: #6b778c;
+}
+
+.footer {
+  padding-top: 40px;
+}
+
+.gmail {
+  display: flex;
+  position: absolute;
+  font-size: 20px;
+  padding-top: 20px;
+  color: #42526e;
+  font-weight: 600;
+  justify-content: center;
+  right: 69px;
 }
```

Comment:
- [ ] Regarding the email address, which is also not centralized, I suggest you remove line 407. Please, fix it and you will get the correct position for this `p` tag.

Reply:
I have implemented the changes